modular_zubbers/code/modules/clothing/belt/sabre.dm

```diff
@@ -0,0 +1,24 @@
+/obj/item/melee/sabre/centcom
```

Comment:
The NT Rep should not be getting the Captain's sword.

Reply:
> The NT Rep should not be getting the Captain's sword.

They aren't? That's for CentCom Commanders?